packages/connect-node-test/src/transports.spec.ts

```diff
@@ -1,110 +0,0 @@
-// Copyright 2021-2024 The Connect Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import { Code, ConnectError } from "@connectrpc/connect";
-import {
-  createConnectTransport,
-  Http2SessionManager,
-} from "@connectrpc/connect-node";
-
-describe("createConnectTransport()", function () {
-  it("should take just httpVersion and baseUrl", function () {
-    const t = createConnectTransport({
-      httpVersion: "2",
-      baseUrl: "https://example.com",
-    });
-    expect(t).toBeDefined();
-  });
-  it("should take session options", function () {
-    const t = createConnectTransport({
-      httpVersion: "2",
-      baseUrl: "https://example.com",
-      pingIntervalMs: 1000 * 30,
-      pingIdleConnection: true,
-      pingTimeoutMs: 1000 * 5,
-      idleConnectionTimeoutMs: 1000 * 60 * 5,
-    });
-    expect(t).toBeDefined();
-  });
-  it("should take node options", function () {
-    const t = createConnectTransport({
-      httpVersion: "2",
-      baseUrl: "https://example.com",
-      nodeOptions: {
-        maxSessionMemory: 1024 * 1024 * 4,
-      },
-    });
-    expect(t).toBeDefined();
-  });
-  it("should take session manager", function () {
-    const sm = new Http2SessionManager(
-      "https://example.com",
-      {
-        pingIntervalMs: 1000 * 10,
-      },
-      {
-        maxSessionMemory: 1024 * 1024 * 4,
-      },
-    );
-    const t = createConnectTransport({
-      httpVersion: "2",
-      baseUrl: "https://example.com",
-      sessionManager: sm,
-    });
-    expect(t).toBeDefined();
-  });
-});
-
-describe("using a session manager to open a connection before starting an application", function () {
```

Comment:
I don't think we test elsewhere that the session manager is wired up correctly from the transport option, so it would be best to keep this.

Reply:
I don't see it testing the transport option in any way, it seems to be just testing the `connect` which is tested in the session manager spec.